openedx/adg/lms/applications/api_views.py

```diff
@@ -0,0 +1,221 @@
+"""
+API views for applications app
+"""
+
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+from django.utils.translation import ugettext_lazy as _
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from rest_framework import status
+from rest_framework.authentication import BasicAuthentication
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Education, WorkExperience
+from .serializers import EducationSerializer, WorkExperienceSerializer
+
+
+class EducationAPIView(APIView):
+    """
+    CRUD implementation of education data
+    """
+    # TODO: BasicAuthentication is added here for testing purposes, pleases remove this when these APIs are integrated.
+    authentication_classes = (BasicAuthentication, SessionAuthenticationAllowInactiveUser,)
+    permission_classes = (IsAuthenticated,)
```

Comment:
Can we use `openedx.core.lib.api.view_utils import view_auth_classes` here for authentication and permissions?
`@view_auth_classes(is_authenticated=True)` above the class